x-pack/plugins/apm/server/routes/fleet/create_cloud_apm_package_policy.ts

```diff
@@ -65,7 +66,7 @@ export async function createCloudApmPackgePolicy({
     savedObjectsClient,
     esClient,
     mergedAPMPackagePolicy,
-    { force: true, bumpRevision: true }
```

Comment:
This ensures the package policy ID is set correct. There was a bug previously in https://github.com/elastic/kibana/pull/128496 .